isis/src/control/objs/BundleUtilities/BundleObservation.h

```diff
@@ -120,20 +133,34 @@ namespace Isis {
      
       LinearAlgebra::Vector &parameterWeights();
       LinearAlgebra::Vector &parameterCorrections();
-//    LinearAlgebra::Vector &parameterSolution();
       LinearAlgebra::Vector &aprioriSigmas();
       LinearAlgebra::Vector &adjustedSigmas();
       
       const BundleObservationSolveSettingsQsp solveSettings();
 
-//    QStringList serialNumbers();
-
       bool applyParameterCorrections(LinearAlgebra::Vector corrections);
       bool initializeExteriorOrientation();
       void initializeBodyRotation();
       void updateBodyRotation();
 
-      QString formatBundleOutputString(bool errorPropagation, bool imageCSV=false);
+      std::vector<double> coefX;
+      std::vector<double> coefY;
+      std::vector<double> coefZ;
+
+      std::vector<double> coefRA;
+      std::vector<double> coefDEC;
+      std::vector<double> coefTWI;
```

Comment:
I might be missing the point here, but why are these instantiated in the header file...?